tchannel-core/src/main/java/com/uber/tchannel/checksum/Checksums.java

```diff
@@ -21,25 +21,25 @@
  */
 package com.uber.tchannel.checksum;
 
-import com.uber.tchannel.messages.CallMessage;
+import com.uber.tchannel.frames.CallFrame;
```

Comment:
why are you renaming all of the messages? If you want to do that put it in its own diff.